src/Slack.php

```diff
@@ -35,19 +35,56 @@ class Slack
      */
     private $config;
 
-    public function __construct(array $config)
+    public function __construct(array $config = [])
```

Comment:
I don't think this is the best way to do this. This class does require the config files and shouldn't be its responsibility to get what it needs (Dependency Inversion Principle). You can already use the code you suggested `Slack::to('#scientists')->webhook('https://hooks.slack.com/services/XXXXXXXXX/XXXXXXXXX/x00XXXXXXXXX')->send(':thinking_face:');` by using the facade and, even shorter, by using the alias defined on composer.json `\Slack`.